apps/dashboard/app/controllers/files_controller.rb

```diff
@@ -52,7 +52,7 @@ def fs
                                         end
 
           if can_download
-            zipname = "#{@path.basename.to_s.gsub('"', '\"')}.zip"
+            zipname = "#{@path.basename}.zip"
             response.set_header 'Content-Disposition', "attachment; filename=\"#{zipname}\""
             response.set_header 'Content-Type', 'application/zip'
```

Comment:
Since `zipname` is getting escaped correctly, are these headers getting overwritten every time we call `send_stream`?

Reply:
Apparently so. I'll look into specifying them in `send_stream` and see if there's any difference.